smac_planner/src/smac_planner.cpp

```diff
@@ -91,7 +93,7 @@ void SmacPlanner::configure(
   std::shared_ptr<nav2_costmap_2d::Costmap2DROS> costmap_ros)
 {
   _node = parent;
-  _costmap = costmap_ros->getCostmap();
+  _original_costmap = costmap_ros->getCostmap();
```

Comment:
This is an updating char * that will update over time, so its not just the original costmap but the costmap itself that is live updating. I think this should be back to the original name

Reply:
Ah! My bad, I thought it remained static throughout execution